tests/example-empty-file/expected_results.json

```diff
@@ -1,5 +1,16 @@
 {
-  "version": 1,
+  "version": 2,
   "status": "fail",
```

Comment:
@ErikSchierboom could I please ask for a review here? In order to support multiple methods in tests, I've moved the method check to individual test cases (earlier there was one check performed per test suite). In this case, if all tests fail (because the method was not found), should the status of the entire test run be "fail" or "error"? I think "fail" is OK, but the documentation of the Test Runner Interface mentions:

```
The following overall statuses are valid:

* pass: All tests passed
* fail: At least one test has the status fail or error
* error: No test was executed correctly (this usually means e.g. a compile error, or a syntax error)
```

Technically, no test was executed correctly here, so perhaps "error" is the right status? But in that case, I'll have to provide a message, which might involve several method names etc., so I would prefer to stick to "fail". I would like to know your opinion, if you don't mind.

Reply:
Right, I can see the confusion. In your case, are the tests all being run? If so, it would be `fail`. The `error` status is just there for when the tests _can't even run_, often due to a syntax error preventing the program to compile.

Reply:
Thanks for clarification :) This is what I hoped for, as it's easier to implement :P